packages/ckeditor5-dev-translations/lib/templates/translation.po

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+#
+#                                     !!! IMPORTANT !!!
+#
+#         Before you edit this file, please keep in mind that contributing to the project
+#                translations is possible ONLY via the Transifex online service.
+#
+#         To submit your translations, visit https://www.transifex.com/ckeditor/ckeditor5.
+#
+#                   To learn more, check out the official contributor's guide:
+#     https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/contributing.html
```

Comment:
```suggestion
# Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
#
# Want to contribute to this file? Submit your changes via a GitHub Pull Request.
#
# Check out the official contributor's guide:
# https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/contributing.html
```

According to https://github.com/cksource/ckeditor5-internal/issues/3802.

Reply:
Done in https://github.com/ckeditor/ckeditor5-dev/pull/1020/commits/eb102f33e86d9f9defe1959fa92f428c73eeb691.